src/helpers/tests/geoChart/common.spec.ts

```diff
@@ -124,4 +125,78 @@ describe("common", () => {
             },
         );
     });
+
+    describe("getGeoPushpinBucketsInMdObject", () => {
+        const attributeGeo1 = {
+            visualizationAttribute: {
+                localIdentifier: "localIdentifier123",
+                displayForm: {
+                    uri: "/gdc/displayform/geo1",
+                },
+            },
+        };
+        const attributeGeo2 = {
+            visualizationAttribute: {
+                localIdentifier: "localIdentifier456",
+                displayForm: {
+                    uri: "/gdc/displayform/geo2",
+                },
+            },
+        };
+
+        it("should return buckets has toottip text when geo pushpin has location item", () => {
+            const mdObject = {
+                visualizationClass: {
+                    uri: "/gdc/visualizationclass/1",
+                },
+                buckets: [
+                    {
+                        localIdentifier: "location",
+                        items: [attributeGeo1],
+                    },
+                ],
+                tooltipText: {
+                    localIdentifier: "tooltipText",
+                    items: [attributeGeo2],
+                },
+            };
+            const buckets = getGeoPushpinBucketsInMdObject(mdObject);
+            expect(buckets).toEqual({
+                ...mdObject,
+                buckets: [
+                    {
+                        localIdentifier: "location",
+                        items: [attributeGeo1],
+                    },
+                    {
+                        localIdentifier: "tooltipText",
+                        items: [attributeGeo2],
+                    },
+                ],
+            });
+        });
+
+        it("should return buckets without toottip text when geo pushpin does not have location item", () => {
+            const mdObject = {
+                visualizationClass: {
+                    uri: "/gdc/visualizationclass/1",
+                },
+                buckets: [
+                    {
+                        localIdentifier: "size",
+                        items: [attributeGeo1],
+                    },
+                ],
+            };
+            const buckets = getGeoPushpinBucketsInMdObject(mdObject);
+            expect(buckets).toEqual(mdObject);
+        });
+    });
+
+    describe("transformPositionGeoConfig", () => {
+        it("should reverse an array", () => {
```

Comment:
how about `should switch position of [lat, lng] to [lng, lat]`?

Reply:
updated